coldfront/core/allocation/templates/secure_dir/secure_dir_request/pi_selection.html

```diff
@@ -28,23 +24,6 @@ <h1>Secure Directory: PI Selection</h1><hr>
 
   <p>Select a PI of the project. This PI will be asked to sign a Researcher Use Agreement during the approval process.</p>
 
-  <form action="" method="post">
-    {% csrf_token %}
```

Comment:
not sure why there wasnt a "first step" and "previous" option here but it it wouldn't hurt to include